### PR TITLE
Reverts #10129 - Cult golem shells can again be finished by anyone

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
This change was really unnecesary, many players agree with me that it only hurt free golems being unable to use brass and runic metals found in lavaland ruins (the ratvar ruin literally has 50 brass for this exact purpose, no clock cultist is ever gonna use that).
The main point of that change was that "cult golems are too OP". Yes, they are good, but they are not the option AGAINST CULTISTS, which is a plain adamantine/metallic-hydrogen golem for it's anti-magic properties and tankiness.
Having an occassion to use said material is so rare, it's what makes it so exciting and fun.

### Why is this change good for the game?
Doesn't buff nor nerf cult, many players agreed that it's lame not being able to make golems out of wild materials found in lavaland and brought to sci by miners.

### Briefly describe your PR and the impacts of it, in layman's terms. 
reverts: https://github.com/yogstation13/Yogstation/pull/10129

### What should players be aware of when it comes to the changes your PR is implementing?
That you can make every golem again.
### What general grouping does this PR fall under? 
revert

# Changelog
:cl:  
rscdel: reverts #10129, you can make every single golem again
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
image

**Comment by maxion12345** [comment]
ngl blood cult golem was just used mostly by powergamers

**Review by JamieD1 — APPROVED** [review]
